src/models/comment.dao.ts

```diff
@@ -1,54 +1,160 @@
 import dataSource from './database';
 
-async function addCommentOnPost(userId: string, postId: string, comment: string) {
-  return await dataSource.query(`
-    INSERT INTO comments(
-      comment,
-      posting_id,
-      user_id
-      )
-    VALUES(?, ?, ?);`
-  , [comment, postId, userId])
-}
-
-async function findCommentById(commentId: string) {
-  const [comment] = await dataSource.query(`
-      SELECT
-        comments.comment,
-        postings.id AS post_id,
-        comments.created_at,
-        comments.updated_at,
-        users.nickname as user_nickname,
-        users.id as user_id
-      FROM comments
-      JOIN users    ON comments.user_id     = users.id
-      JOIN postings ON comments.posting_id  = postings.id
-      WHERE comments.id = ?`
-      , [commentId]
-    );
-  return comment as {comment: string, poast_id: string, created_at: string, updated_at: string, user_nickname: string, user_id: string, };
-}
-
-async function updateComment(commentId: string, comment: string) {
+const addCommentOnPost = async (userId: number, postId: number, comment: string, is_secret: number) => {
   await dataSource.query(`
-      UPDATE comments
-        SET comment = ?
-      WHERE id = ?`
-      , [comment, commentId]
+    INSERT INTO comments (
+      users_id,
+      company_posts_id,
+      comment_content,
+      depth,
+      sequence,
+      is_secret
     )
+    VALUES (?, ?, ?, ?, ?, ?)`
+  , [userId, postId, comment, 1, 1, is_secret])
```

Comment:
const commentDepth = {
  FIRST : 1,
  SECOND : 2
} 를 추가해주세요

```suggestion
  , [userId, postId, comment, commentDepth.FIRST, 1, is_secret])
```

Reply:
반영했습니다. 감사합니다~!